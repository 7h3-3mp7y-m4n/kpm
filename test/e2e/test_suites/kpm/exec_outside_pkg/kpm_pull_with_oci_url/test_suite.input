kpm pull oci://localhost:5001/test/k8s