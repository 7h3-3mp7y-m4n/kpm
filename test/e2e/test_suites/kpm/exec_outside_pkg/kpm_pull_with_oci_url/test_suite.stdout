start to pull 'oci://localhost:5001/test/k8s'
the lastest version '1.27' will be pulled
pulling '/test/k8s:1.27' from 'localhost:5001/test/k8s'
pulled 'oci://localhost:5001/test/k8s' in '<workspace>/localhost:5001/test/k8s' successfully